edb/testbase/server.py

```diff
@@ -1257,6 +1257,17 @@ async def squery(self, query):
         ''')
         return await self.scon.fetch(rewritten)
 
+    async def squery_values(self, query):
+        res = await self.squery(query)
+        return [list(r.values()) for r in res]
+
+    def assertShape(self, res, rows, cols, column_names=None):
```

Comment:
Add a docstring for this

I would also say call it `assert_shape`, except that `assertShape` matches other assert methods. Your call.